sda-validator/orchestrator/api/api_test.go

```diff
@@ -0,0 +1,617 @@
+						Result:   "passed",
+						Messages: nil,
+					},
+				},
+			},
+		},
+	}
+
+	ts.mockDatabase.On("ReadValidationResult", testValidationResult.ValidationID, (*string)(nil)).Return(testValidationResult, nil)
+
+	w := httptest.NewRecorder()
+
+	req, _ := http.NewRequest("GET", fmt.Sprintf("/admin/result?validation_id=%s", testValidationResult.ValidationID), nil)
+	ts.ginEngine.ServeHTTP(w, req)
+
+	ts.Equal(http.StatusOK, w.Code)
+
+	var resultResponse []openapi.ResultResponseInner
+	if err := json.Unmarshal(w.Body.Bytes(), &resultResponse); err != nil {
+		ts.FailNow(err.Error(), "failed to parse response body to []openapi.ResultResponseInner")
+	}
+
+	ts.mockDatabase.AssertNumberOfCalls(ts.T(), "ReadValidationResult", 1)
+	ts.mockDatabase.AssertCalled(ts.T(), "ReadValidationResult", testValidationResult.ValidationID, (*string)(nil))
+
+	ts.Equal(len(testValidationResult.ValidatorResults), len(resultResponse))
+
+	ts.Equal(testValidationResult.ValidatorResults[0].Result, resultResponse[0].Result)
+	ts.Equal(testValidationResult.ValidatorResults[0].ValidatorID, resultResponse[0].ValidatorId)
+	ts.Equal(testValidationResult.ValidatorResults[0].StartedAt, resultResponse[0].StartedAt)
+	ts.Equal(testValidationResult.ValidatorResults[0].FinishedAt, resultResponse[0].FinishedAt)
+	ts.Equal(len(testValidationResult.ValidatorResults[0].Messages), len(resultResponse[0].Messages))
+	ts.Equal(len(testValidationResult.ValidatorResults[0].Files), len(resultResponse[0].Files))
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[0].Result, resultResponse[0].Files[0].Result)
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[0].FilePath, resultResponse[0].Files[0].Path)
+	ts.Equal(len(testValidationResult.ValidatorResults[0].Files[0].Messages), len(resultResponse[0].Files[0].Messages))
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[1].Result, resultResponse[0].Files[1].Result)
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[1].FilePath, resultResponse[0].Files[1].Path)
+	ts.Equal(len(testValidationResult.ValidatorResults[0].Files[1].Messages), len(resultResponse[0].Files[1].Messages))
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[2].Result, resultResponse[0].Files[2].Result)
+	ts.Equal(testValidationResult.ValidatorResults[0].Files[2].FilePath, resultResponse[0].Files[2].Path)
+	ts.Equal(len(testValidationResult.ValidatorResults[0].Files[2].Messages), len(resultResponse[0].Files[2].Messages))
+}
+
+func (ts *ValidatorAPITestSuite) TestResultGet_NoTokenInContext() {
+	ginEngine := openapi.NewRouter(openapi.ApiHandleFunctions{
+		ValidatorOrchestratorAPI: &validatorAPIImpl{
+			sdaAPIURL:                     ts.httpTestServer.URL,
+			sdaAPIToken:                   "mock-sdaAPIToken",
+			validationFileSizeLimit:       1024 * 4,
+			validationJobPreparationQueue: "job-preparation-queue",
+			broker:                        ts.mockBroker,
+		}})
+
+	ts.ginEngine.Use(func(c *gin.Context) {
```

Comment:
```suggestion
	ginEngine.Use(func(c *gin.Context) {
```

Otherwise, the middleware will not be applied to `ginEngine`